posts/2025-03-02-Rustup-1.28.0.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "Announcing Rustup 1.28.0"
+author: The Rustup Team
+---
+
+The rustup team is happy to announce the release of rustup version 1.28.0.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.28.0 is as easy as stopping any programs which may be using Rustup (e.g. closing your IDE) and running:
```

Comment:
Suggest we do "how to upgrade" after "what's new".

Reply:
@djc This is pre-existing, and I believe it's there to be coherent with Rust's release announcements such as https://blog.rust-lang.org/2025/01/09/Rust-1.84.0.html.